src/seedu/addressbook/ui/TextUi.java

```diff
@@ -41,6 +37,7 @@
 
     private final Scanner in;
     private final PrintStream out;
+    private final Formatter formatter = new Formatter(this);
```

Comment:
This is not really necessary. The `TextUI` can receive formatting related stuff and do the I/O